src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -519,6 +518,28 @@ export class BrowsingContextImpl {
     };
   }
 
+  async reload(
+    ignoreCache: boolean,
+    wait: BrowsingContext.ReadinessState
+  ): Promise<Message.EmptyResult> {
+    await this.#cdpTarget.cdpClient.sendCommand('Page.reload', {
+      ignoreCache,
```

Comment:
To test it properly we need a local server. Than we could make a page returning current timestamp each time it is loaded, and after reload verify the content has changed or not.

The local server is not setup for e2e, but it is for WPT, so I guess we can test it from WPT, as from our perspective it's only 1 flag.

Reply:
Ack